modules/consensus/persist.go

```diff
@@ -84,7 +97,7 @@ func (cs *ConsensusSet) loadDiffs() {
 		// Blocks loaded from disk are trusted, don't bother with verification.
 		lockID := cs.mu.Lock()
 		// This guard is for when the program is stopped. It is temporary.
-		// DEPRICATED
+		// DEPRECATED
 		if !cs.db.open {
 			break
 		}
```

Comment:
When every structure is loaded from disk immediately, there will be a single write lock on consensus when loading and the database can't be closed while a write lock is held. We'll probably need this guard in gateway though.